src/Data/Uniques/bow.lua

```diff
@@ -73,48 +73,74 @@ Variant: Minion Attack and Cast Speed (Pre 3.17.0)
-{variant:9,23}{crafted}(7-13)% increased Cast Speed
-{variant:9,23}{crafted}10% chance to gain Arcane Surge when you Kill an Enemy
-{variant:10,24}{crafted}Minions have (16-28)% increased Attack Speed
-{variant:10,24}{crafted}Minions have (16-28)% increased Cast Speed
-{variant:11,25}{crafted}(4-12)% chance to deal Double Damage
-{variant:12,26}{crafted}(13-36)% chance to deal Double Damage while Focused
-{variant:13,27}{crafted}Trigger a Socketed Spell when you Use a Skill, with a 8 second Cooldown
+{variant:1}{crafted}+(18-45)% Critical Strike Multiplier while there is a Rare or Unique Enemy Nearby
+{variant:2}{crafted}(11-22)% increased Attack Speed while a Rare or Unique Enemy is Nearby
+{variant:3}{crafted}(5-6)% increased Damage per Power Charge
+{variant:4}{crafted}(5-6)% increased Damage per Frenzy Charge
+{variant:5}{crafted}(5-6)% increased Damage per Endurance Charge
+{variant:6}{crafted}+(30-250) to Accuracy Rating
+{variant:7}{crafted}(8-16)% increased Attack Speed
+{variant:6,7}{crafted}+(7-18)% to Quality
+{variant:8}{crafted}(8-16)% increased Attack Speed
+{variant:8}{crafted}10% chance to Trigger Level 1 Blood Rage when you Kill an Enemy
+{variant:9}{crafted}(7-13)% increased Cast Speed
+{variant:9}{crafted}10% chance to gain Arcane Surge when you Kill an Enemy
+{variant:10}{crafted}Minions have (16-28)% increased Attack Speed
+{variant:10}{crafted}Minions have (16-28)% increased Cast Speed
+{variant:11}{crafted}(4-12)% chance to deal Double Damage
+{variant:12}{crafted}(13-36)% chance to deal Double Damage while Focused
+{variant:13}{crafted}Trigger a Socketed Spell when you Use a Skill, with a 8 second Cooldown
+{variant:15}+(54-60)% Critical Strike Multiplier while there is a Rare or Unique Enemy Nearby
+{variant:16}(27-30)% increased Attack Speed while a Rare or Unique Enemy is Nearby
+{variant:17}(7-8)% increased Damage per Power Charge
+{variant:18}(7-8)% increased Damage per Frenzy Charge
+{variant:19}(7-8)% increased Damage per Endurance Charge
+{variant:20}+(311-350) to Accuracy Rating
+{variant:20}+(25-28) to Strength and Dexterity
+{variant:21}(18-22)% increased Attack Speed
+{variant:21}+(25-28) to Dexterity and Intelligence
+{variant:22}(28-32)% increased Critical Strike Chance
+{variant:22}+(25-28) to Strength and Intelligence
+{variant:23}(26-31)% increased Cast Speed
+{variant:23}15% chance to gain Arcane Surge when you Kill an Enemy
+{variant:24}Minions have (34-38)% increased Attack Speed
+{variant:24}Minions have (34-38)% increased Cast Speed
+{variant:25}(12-14)% chance to deal Double Damage
+{variant:26}(36-40)% chance to deal Double Damage while Focused
+{variant:27}Trigger a Socketed Spell when you Use a Skill, with a 4 second Cooldown
+{variant:27}Spells Triggered this way have 150% more Cost
+{variant:28}+(24-28)% to Fire Damage over Time Multiplier
+{variant:29}+(24-28)% to Physical Damage over Time Multiplier
+{variant:30}+(24-28)% to Chaos Damage over Time Multiplier
```

Comment:
These should also have `{crafted}` in front of them

Reply:
I've totally forgotten how to add to someone else's PR in GitHub Desktop... is dropping them here helpful at all?
```lua
{variant:15}{crafted}+(54-60)% Critical Strike Multiplier while there is a Rare or Unique Enemy Nearby
{variant:16}{crafted}(27-30)% increased Attack Speed while a Rare or Unique Enemy is Nearby
{variant:17}{crafted}(7-8)% increased Damage per Power Charge
{variant:18}{crafted}(7-8)% increased Damage per Frenzy Charge
{variant:19}{crafted}(7-8)% increased Damage per Endurance Charge
{variant:20}{crafted}+(311-350) to Accuracy Rating
{variant:20}{crafted}+(25-28) to Strength and Dexterity
{variant:21}{crafted}(18-22)% increased Attack Speed
{variant:21}{crafted}+(25-28) to Dexterity and Intelligence
{variant:22}{crafted}(28-32)% increased Critical Strike Chance
{variant:22}{crafted}+(25-28) to Strength and Intelligence
{variant:23}{crafted}(26-31)% increased Cast Speed
{variant:23}{crafted}15% chance to gain Arcane Surge when you Kill an Enemy
{variant:24}{crafted}Minions have (34-38)% increased Attack Speed
{variant:24}{crafted}Minions have (34-38)% increased Cast Speed
{variant:25}{crafted}(12-14)% chance to deal Double Damage
{variant:26}{crafted}(36-40)% chance to deal Double Damage while Focused
{variant:27}{crafted}Trigger a Socketed Spell when you Use a Skill, with a 4 second Cooldown
{variant:27}{crafted}Spells Triggered this way have 150% more Cost
{variant:28}{crafted}+(24-28)% to Fire Damage over Time Multiplier
{variant:29}{crafted}+(24-28)% to Physical Damage over Time Multiplier
{variant:30}{crafted}+(24-28)% to Chaos Damage over Time Multiplier

```

Reply:
This was helpful, thanks!